clients/instance/ibm-pi-volume.go

```diff
@@ -44,6 +45,10 @@ func (f *IBMPIVolumeClient) GetAll() (*models.Volumes, error) {
 	params := p_cloud_volumes.NewPcloudCloudinstancesVolumesGetallParams().
 		WithContext(f.ctx).WithTimeout(helpers.PIGetTimeOut).
 		WithCloudInstanceID(f.cloudInstanceID)
+	// Check for satellite differences in this endpoint
+	if strings.Contains(f.session.Options.Zone, helpers.PIStratosRegionPrefix) && (params.Auxiliary != nil && params.ReplicationEnabled != nil) {
+		return nil, fmt.Errorf("replication enabled parameter is not supported in satellite location, check documentation")
```

Comment:
do we also need to add `auxiliary` param as well in the error message?

Reply:
Yes, that is a typo. Made changes.